rotkehlchen/accounting/structures/processed_event.py

```diff
@@ -77,22 +86,22 @@ def to_exported_dict(
             'pnl_free': str(self.pnl.free),
         }
         tx_hash = self.extra_data.get('tx_hash', None)
-        if eth_explorer:
+        if export_type == AccountingEventExportType.CSV:
             taxable_basis = free_basis = ''
             if self.cost_basis is not None:
                 taxable_basis, free_basis = self.cost_basis.to_string(ts_converter)
             exported_dict['cost_basis_taxable'] = taxable_basis
             exported_dict['cost_basis_free'] = free_basis
             exported_dict['asset'] = str(self.asset)
-            if tx_hash:
-                exported_dict['notes'] = f'{eth_explorer}{tx_hash}  ->  {self.notes}'
-        else:
+            if tx_hash is not None and evm_explorer is not None:
```

Comment:
Shouldn't it always be given?

```suggestion
            if tx_hash is not None:
```